packages/server/billing/helpers/teamLimitsCheck.ts

```diff
@@ -172,37 +163,18 @@ export const checkTeamsLimit = async (orgId: string, dataLoader: DataLoaderWorke
   const billingLeadersIds = billingLeaders.map((billingLeader) => billingLeader.id)
 
   if (organization.activeDomain) {
-    await enableUsageStats(billingLeadersIds, orgId)
-    await sendWebsiteNotifications(orgId, billingLeadersIds, dataLoader)
-  }
-
-  await scheduleJobs(scheduledLockAt, orgId)
-}
-
-export const processLockOrganizationJob = async (
-  job: ScheduledJobOrganizationLock,
-  dataLoader: DataLoaderWorker
-) => {
-  const {orgId, runAt} = job
-
-  const organization = await dataLoader.get('organizations').load(orgId)
-
-  // Skip the job if unlocked or already locked or scheduled lock date changed
-  if (
-    !organization.scheduledLockAt ||
-    organization.lockedAt ||
-    organization.scheduledLockAt.getTime() !== runAt.getTime()
-  ) {
-    return
+    await Promise.all([
+      enableUsageStats(billingLeadersIds, orgId),
+      sendWebsiteNotifications(orgId, billingLeadersIds, dataLoader),
+      billingLeaders.map((billingLeader) =>
+        sendTeamsLimitEmail({
+          user: billingLeader,
+          orgId,
+          orgName,
+          emailType: 'thirtyDayWarning'
+        })
+      )
+    ]),
+      scheduleTeamLimitsJobs(scheduledLockAt, orgId)
```

Comment:
Should it be outside of Promise.all?

Reply:
Why is that?

Reply:
@nickoferrall I mean it is currently outside of `Promise.all`, should we put it inside?

Reply:
Oh thanks! I thought it was already included. Update pushed